src/Core/Plugin.php

```diff
@@ -30,7 +30,7 @@ class Plugin
      *
      * @var \Cake\Core\PluginCollection|null
      */
-    protected static $plugins;
+    protected static ?\Cake\Core\PluginCollection $plugins = null;
```

Comment:
Should just use classname here. It's in the same namespace so even a `use` statement isn't required. Too bad such changes will have to be done manually.

Reply:
There is a sniff I used to fix these up when generating the parameter/return type hints, but have to manually enable it.